packages/shared/package.json

```diff
@@ -37,12 +37,14 @@
         "moment": "^2.29.2"
     },
     "dependencies": {
+        "@deriv/translations": "^1.0.0",
```

Comment:
we need for many functions that we are using in the stores of core

Reply:
Adding translations as a dependency is shared is not recommended as shared is only supposed to have simple utility functions. Can we avoid this?

Reply:
we need this translation in some of our utilities which are using in the core, trader and reports packages. removing them from shared and move those functions to each package will causes in duplicating multiple files.